app/Importers/BirdDataImporter.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Importers;
+
+use App\Models\City;
+use App\Models\CityAlternativeName;
+use App\Models\Country;
+use App\Services\MapboxGeocodingService;
+use Throwable;
+
+class BirdDataImporter extends DataImporter
+{
+    private const PROVIDER_ID = 1;
+
+    protected string $fetchedData;
+
+    public function extract(): static
+    {
+        try {
+            $html = file_get_contents("https://www.bird.co/map/");
```

Comment:
How you will test it (I mean this importer class)?
Getting html from the outside should be moved to some command or even class. Then you will be able to inject mocked content.
Right now if you use this importer in tests, it will always get content from real site.

Reply:
Can we make issue about it?

Reply:
Yes, it will require to refactor all importers.
https://github.com/blumilksoftware/escooters/issues/83